libdevcore/Log.cpp

```diff
@@ -77,7 +78,13 @@ LogOutputStreamBase::LogOutputStreamBase(char const* _id, std::type_info const*
 			static char const* c_sep1 = EthReset EthBlack "|" EthNavy;
 			static char const* c_sep2 = EthReset EthBlack "|" EthTeal;
 			static char const* c_end = EthReset "  ";
-			m_sstr << _id << c_begin << buf << c_sep1 << std::left << std::setw(8) << getThreadName() << c_sep2 << c_end;
+			if (g_syslog) 
```

Comment:
If you're not going to output the time, why bother retrieving it and formatting it above?

Reply:
You're right, that can be moved into the 'else'.